compose/gettingstarted.md

```diff
@@ -90,6 +90,7 @@ following:
     COPY requirements.txt requirements.txt
     RUN pip install -r requirements.txt
     COPY . .
+    EXPOSE 5000
```

Comment:
expose should not be needed, as it's only adding metadata to the image

Reply:
Flask app is running at port 5000, docker-compose maps 5000 port in the docker container to 5000 post on the host machine. Doesn't the docker container need to expose the 5000 port? similar to (https://docs.docker.com/get-started/part2/#sample-dockerfile)

Reply:
Thank you @cerofrais. I've discussed with @thaJeztah. We also need to add a description of 'Expose'. I'll make this change in an incoming PR.